src/fortran/fct.cpp

```diff
@@ -614,6 +621,17 @@ extern "C" {
         return 0;
     }
 
+    status_t th_getgibbs_rt_(const integer* n,
+                             doublereal* grt) {
+      try {
+        thermo_t* thrm = _fth(n);
+        thrm->getGibbs_RT(grt);
```

Comment:
The `thermo_t` typedef is deprecated. You can write this more compactly as just
```suggestion
        _fth(n)->getGibbs_RT(grt);
```

Reply:
Should the more compact way of writing it be used even though the other routines that are similar do not do so, for example in `th_getenthalpies_rt` just prior it has:

```
ThermoPhase* therm = _fth(n);
therm->getEnthalpy_RT(h_rt);
```

I am happy to update existing routines to make them use the compact style if that is the preferred approach, or to use the 2-line style in this routine to be consistent with the others.

Reply:
I got rid of `thermo_t` but left it as the 2-line style to conform to the similar routines around it -- I was hesitant to update all the routines since there is not any testing (that I can find) for the Fortran interface, so I didn't want to unintentionally break something. If you'd like me to update to the compact style for this routine, and/or if you'd like them all updated, let me know.